File: CollegeManagementSystem/CollegeManagementSystem/ViewController.m
//
//  ViewController.m
//  CollegeManagementSystem
//
//  Created by apple on 2019/3/26.
//  Copyright © 2019 Chauncey Qu. All rights reserved.
//

#import "ViewController.h"
#import "ObjectModel/MyClass+CoreDataClass.h"
#import "AppDelegate.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    AppDelegate* appDelegate = (AppDelegate *)[UIApplication sharedApplication].delegate;
    
    NSMutableArray* arrMyClasses = [[NSMutableArray alloc] init];
    NSArray* arrMyClassesName = @[@"99级1班",@"99级2班",@"99级3班"];
    for (NSString* className in arrMyClassesName) {
        MyClass* newMyClass = [NSEntityDescription insertNewObjectForEntityForName:@"MyClass" inManagedObjectContext:appDelegate.persistentContainer.viewContext];
        newMyClass.name = className;
        [arrMyClasses addObject:newMyClass];
    }
}


@end

File: SwiftSDKTestDemo/SwiftSDKTestDemo/ViewController.swift
//
//  ViewController.swift
//  SwiftSDKTestDemo
//
//  Created by apple on 2019/3/10.
//  Copyright © 2019 Chauncey Qu. All rights reserved.
//

import UIKit
import SwiftSDK
class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        var test = SwiftSDKClass()
        test.getName()
    }


}

